_includes/title.html

```diff
@@ -19,10 +19,14 @@
     <h1>
         {{ page.title | xml_escape }}
         <small>{{ page.subtitle }}</small>
+        
         {% if edit_path %}
             <a class="btn btn-primary pull-right" href="{{ edit_path }}">Edit on Github</a>
         {% elsif show_news_link %}
             <a class="btn btn-primary pull-right" href="/news/atom.xml">Subscribe via Atom</a>
         {% endif %}
     </h1>
+    {% if page.author %}
```

Comment:
I personally find that authorship is good for two reasons:
- people can find you on github for questions. Tutorials will probably expire some day. And I won't know until someone asks me.
- I think it is motivating. It is some kind of recognition. You see it a lot in the modding community. Every contributor's two minutes of fame. :) I was thankful to @rhempel that he mentioned me in the ev3dev-lang-python source.

I believe more (potential) contributors are like me in the above cases. Let's motivate them!